src/util/updateNetworks.js

```diff
@@ -0,0 +1,27 @@
+import { rpc, settings } from '@cityofzion/neon-js';
+import { keys } from 'lodash';
+
+import { NOS_LOCAL, PREDEFINED_NETWORKS } from '../values/networks';
+
+const { networks } = settings;
+
+const nosLocalConfig = {
+  name: NOS_LOCAL,
+  extra: {
+    neoscan: 'http://localhost:4000/api/main_net'
+  }
+};
+
+export default function updateNetworks(userNetworks) {
+  keys(networks).forEach((name) => {
+    if (!PREDEFINED_NETWORKS.includes(name)) {
+      settings.removeNetwork(name);
```

Comment:
Just for my understanding, would this code never get called because the definition of `PREDEFINED_NETWOKRS` is `keys(settings.networks).concat([NOS_LOCAL]);`?

Reply:
`settings.networks` will include neon-js's hardcoded networks (MainNet, TestNet, CozNet), plus anything we added to it via `settings.addNetwork` (nOSLocal & user-defined networks).  The goal here is to remove just user-defined networks, but preserve MainNet, TestNet, CozNet, & nOSLocal.